### PR TITLE
Health page edits

### PR DESCRIPTION
**Pull request by rkelson**
.

### CODE REVIEW

File: data/health/health-small-statistical-areas/index.html [hunk 1]
```diff
@@ -5,45 +5,58 @@
...
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="2018SmallHealthStatisticalAreas">2020 Small Health Statistical Areas</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/SmallStatAreas_2009_small.png %}" alt="Small Health Areas" />
+    <p>"Utah Small Health Statistics Areas" was developed by the Utah Department of Health by combining neighboring ZIP code areas to create 104 small areas ranging in population of approximately 8,000 to 86,000 persons. These boundaries were created to allow for more meaningful analysis by reducing areas to the smallest unit where there is enough data to be reliable and the area is similar. </p>
```
Comment (steveoh) [inline data/health/health-small-statistical-areas/index.html:54]: **nitpick**: remove quotes
```suggestion
    <p>Utah Small Health Statistics Areas was developed by the Utah Department of Health by combining neighboring ZIP code areas to create 104 small areas ranging in population of approximately 8,000 to 86,000 persons. These boundaries were created to allow for more meaningful analysis by reducing areas to the smallest unit where there is enough data to be reliable and the area is similar.</p>
```

**Review by steveoh — COMMENTED** [review]
I'm going to pull out your last two commits and put them in another PR that we won't merge until the sgid10 deprecation.

File: data/health/health-small-statistical-areas/index.html [hunk 2]
```diff
@@ -5,45 +5,58 @@
...
+    item_id: 6fe09518dc62415d88570dfd59fbb4c5
   updates:
     - November 2017
 2014SmallHealthStatisticalAreas:
   downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mYzR2TEdaTHIwa0E&export=download
-      label: '2014 Small Statistical Areas: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mNlNPZU5SRVFJcVU&export=download
-      label: '2014 Small Statistical Areas: Shapefile'
+    - url: https://utah.maps.arcgis.com/home/item.html?id=30c1368e89314f34a547779772cb1d13
```
Comment (steveoh) [inline data/health/health-small-statistical-areas/index.html:29]: **question**: ok these must be shelved or static and not in open data. is that why you kept the download include?

File: data/health/health-small-statistical-areas/index.html [hunk 3]
```diff
@@ -5,45 +5,58 @@
   display_name: JP
   email: agrc@utah.gov
 date: 2011-11-21 17:25:17 -0700
-title: Small Health Statistical Areas
 categories: []
+2020SmallHealthStatisticalAreas:
```
Comment (steveoh) [inline data/health/health-small-statistical-areas/index.html:9]: **thought**: i wonder if we should only keep the 2020 as a main level with the prior years as additional information. The metadata would then only be managed in arcgis online. what do you think?

File: data/health/health-small-statistical-areas/index.html (hunk 2 above)
Comment (rkelson) [reply data/health/health-small-statistical-areas/index.html:29]: yes 2014 is shelved

File: data/health/health-small-statistical-areas/index.html (hunk 3 above)
Comment (rkelson) [reply data/health/health-small-statistical-areas/index.html:9]: yeah I am not too excited about hammering into someone else's pages of data I know nothing about. I think we should merge what I have and revisit the page later, I know it needs more work.

**Pull request merged**